mrp_bom_enforcement/models/mrp_production.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+from odoo.exceptions import ValidationError
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    def action_confirm(self):
+        error_messages = []
+        for production in self:
```

Comment:
```suggestion
        for production in self:
            if not self._force_bom:
                return super().action_confirm()
```

For extendability, can we do like this upon adding the `_force_bom()` method which returns True/False based on a config parameter?